src/index.js

```diff
@@ -1,24 +1,71 @@
 /* eslint-disable no-unused-vars */
-import _ from 'lodash';
 import './style.css';
-import Storage from './modules/storage.js';
-import Handler from './modules/complete.js';
-
-const { handleChangeInCheckbox } = Handler;
-const { allTasks: tasksToDo } = Storage;
-
-const taskCollection = () => {
-  const taskUL = document.getElementById('tasks_id');
-
-  tasksToDo.forEach((task) => {
-    taskUL.insertAdjacentHTML('beforeend', ` <li class="task_list">
-                        <input type="checkbox" class="checkbox_style" data-id=${task.id}  name="${task.description}" ${task.completed ? 'checked' : ''}>
-                        <label for="${task.description}">${task.description}</label><hr>
-                       </li>`);
-  });
-};
+import Data from './data.js';
+import Handlers from './handlers.js';
+
+const { allTasks: tasks } = Data;
+const {
+  handleCheckBoxChange, handleAddTask, removeTask, clearCompleted, createUpdateHandle, updateTask,
+} = Handlers;
+
+const appHeader = `
+<li class="appHeader">
+    <h1>Today's To Do</h1>
+    <span>&#8634</span>
+</li>`;
+
+const input = `
+<li>
+    
+    <input type="text" id="task-description" name="description" placeholder="Add to your list...">
+     
+    <span class="save-task">&crarr;</span>
+</li>
+`;
+
+const todoList = tasks
+  .map((task, index) => `
+    <li class="task-item"> 
+        <p>
+            <span>
+                  <label for="task list"> 
+                  <input class="status" data-index="${index}" type="checkbox"${task.completed ? 'checked' : ''} />
+                     <span  class="description"> ${task.description}  </span>
+                  </label>
+            </span>
+            
+        </p>
+        <p>
+          <i data-index="${index}" class="remove-item">&times</i>
+          <span class="drag-around">&#8942</span>
+        </p>
+    </li>
+`);
+
+const footer = `
+<li id="clear-all">
+    <div>
+        Clear all completed
+    </div>
+</li>
+`;
+
+const minimalist = document.getElementById('minimalist');
+minimalist.innerHTML = `
+    ${appHeader}
+    ${input}
+    ${todoList.join('')}
+    ${footer}
+`;
+
 document.addEventListener('DOMContentLoaded', () => {
-  taskCollection();
-  const checkBoxes = document.querySelectorAll('.checkbox_style');
-  checkBoxes.forEach((checkBox) => checkBox.addEventListener(('change'), (e) => handleChangeInCheckbox(e)));
-});
+  minimalist.addEventListener('change', (e) => handleCheckBoxChange(e));
+  minimalist.addEventListener('click', (e) => removeTask(e));
+  minimalist.addEventListener('click', (e) => createUpdateHandle(e));
+
+  const saveTaskButton = document.querySelector('.save-task');
+  saveTaskButton.addEventListener('click', (e) => handleAddTask(e, false));
```

Comment:
[OPTIONAL] If instead of listening for a button when a task it's being added, we listen for submission in a form we will automatically allow the user to submit using the key `enter`. This results in a better user experience. [You can read more about here.](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event)

Tip: Since forms usually trigger an HTTP request, we can prevent this behavior by calling `event.preventDefault()` in the callback function.

Reply:
I will ensure to add this in my free time. Thank you so much